src/schema/v2/homeView/HomeViewComponent.ts

```diff
@@ -99,5 +136,9 @@ export const HomeViewComponent = new GraphQLObjectType({
         }
       },
     },
+    behaviors: {
```

Comment:
Following the agreed on schema here https://www.notion.so/artsy/Art-Advisor-Dynamic-Home-View-3615106ef393433a9f16dcf13b2414f2#682168e6cf9f47b9ad3cfbe1400a7005

Reply:
Seems like PR ticket might cover https://artsyproduct.atlassian.net/browse/ONYX-1157 as well. 

With that in mind, I left some related questions in my other comment.